packages/api-react/src/services/wallet.ts

```diff
@@ -600,6 +601,9 @@ export const walletApi = apiWithTag.injectEndpoints({
       }),
       invalidatesTags: (_result, _error, { fingerprint }) => [
         { type: 'Keys', id: fingerprint },
+        { type: 'Keys', id: 'LIST' },
+        { type: 'DaemonKey', id: fingerprint },
+        { type: 'DaemonKey', id: 'LIST' },
```

Comment:
If these are necessary, should the same be applied to the `addKey` and `deleteAllKeys` mutations?

Reply:
Nice catch, updated.